File: Server/Server/Server/Model/DTO/LoginUserDTO.cs
namespace Server.Model.DTO
{
    /// <summary>
    /// Représente les informations d'un utilisateur pour la connexion.
    /// </summary>
    public class LoginUserDTO
    {
        private string username;
        private string password;

        /// <summary>
        /// Le nom d'utilisateur de la connexion.
        /// </summary>
        public string Username { get => username; set => username = value; }

        /// <summary>
        /// Le mot de passe associé à l'utilisateur.
        /// </summary>
        public string Password { get => password; set => password = value; }
    }
}

File: Client/src/app/Model/DTO/LoginUserDTO.ts
/**
 * Gère les attributs de l'objet de connexion utilisateur (DTO = Data Transfer Object) sert d'objet de transfert de 
 * données entre un client et un serveur
 */
export class LoginUserDTO {
    private username: string;
    private password: string;
  
    constructor(username: string, password: string) {
      this.username = username;
      this.password = password;
    }
    
    /**
   * Obtient le nom d'utilisateur. utilie pour les test
   * @returns Le nom d'utilisateur
   */
  public get Username(): string {
    return this.username;
  }

}
  